### DISCUSSION TITLE
Recurrent Neural networks

### DISCUSSION THREAD

**Discussion by MehdiRh17** [fastmachinelearning/hls4ml#652] [General]
Hello. I have read a paper about  implementing recurrent neural networks with HLS4ML but I cannot find any example/ code or tutorial on that on the documentation. Can someone please send me a link? Thanks

**Comment by olaiya** [comment]
Likewise, I have also read about RNNs in the context of hls4ml. Is there anything in development that is available to try? It would be really interesting if there was.

**Comment by jmduarte** [comment]
This has now been merged. Take a look at our tests for an example: https://github.com/fastmachinelearning/hls4ml/blob/main/test/pytest/test_rnn.py
